mappings/notes.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "v1beta1",
+  "version": "v1beta2",
```

Comment:
Bumping the version for notes and projects in order to trigger migrations for existing instances of the latest release. The migrations will handle creating the aliases on the new indexes, otherwise querying would fail.